client/jetpack-connect/remote-credentials.js

```diff
@@ -168,8 +172,8 @@ export class OrgCredentialsForm extends Component {
 		) {
 			return ACTIVATION_FAILURE;
 		}
-		if ( installError === 'LOGIN_FAILURE' ) {
```

Comment:
I'd keep it as a comment above the UNKNOWN error case, for error coverage reference.